app/views/layouts/_seo.html.erb

```diff
@@ -0,0 +1,50 @@
+<%# This belongs inside the <head> tag %>
+
+<link rel="publisher" href="https://plus.google.com/u/0/+IndentLabs" />
+
+<%=
+  # Make sure description is no longer than 155 characters
+  site_name = "Notebook"
+  site_description = "Notebook is a set of tools for writers, game designers, and roleplayers to create magnificent universes, and everything within them."
+  site_url = request.host
+  site_image = "http://www.notebook.ai/assets/card-headers/hero-d5161eb41a02535f6656af83cacbdb8c.jpg"
+
+  display_meta_tags description: site_description,
+  # Recommended keywords tag length: up to 255 characters, 20 words.
+  keywords: %w[writing author fiction character universe location nanowrimo],
+  canonical: site_url,
+  twitter: {
+    card: "summary_large_image",
+    title: site_name,
```

Comment:
It will. I'll have to do some work across most views to set those up, and I'll have to learn a little bit more about the gem in order to use it most effectively, but there are a lot of options for setting page titles. I'll work on that more tomorrow.